### PR TITLE
Make adjustments to make the local environment runnable and README tidy up

### PR DESCRIPTION
**Pull request by tm-kn**
Fixed the tox -e interactive command.

And a few changes to tox.ini.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Now you can run tests as shown below:
 tox
 ```
 
-or, you can run them for a specific environment `tox -e python3.8-django3.2-wagtail2.15` or specific test
-`tox -e python3.9-django3.2-wagtail2.15-sqlite wagtail-vector-index.tests.test_file.TestClass.test_method`
+or, you can run them for a specific environment `tox -e py3.11-django4.2-wagtail5.2` or specific test
+`tox -e py3.11-django4.2-wagtail5.2 wagtail-vector-index.tests.test_file.TestClass.test_method`
```
Comment (zerolab) [inline README.md:80]:
```suggestion
`tox -e py3.11-django4.2-wagtail5.2 -- wagtail-vector-index.tests.test_file.TestClass.test_method`
```

File: pyproject.toml [hunk 2]
```diff
@@ -58,7 +58,7 @@ docs = [
 ]
 
 [project.urls]
-Home = "https://github.com/tomusher/wagtail-vector-index"
+Home = "https://github.com/wagtail/wagtail-vector-index"
```
Comment (zerolab) [inline pyproject.toml:61]: 👌🏼

Comment (zerolab) [reply pyproject.toml:61]: Saying that, can update the references in README.md too?

File: tox.ini [hunk 3]
```diff
@@ -58,6 +64,8 @@ basepython = python3.11
 deps =
     wagtail>=5.2
 
+    .[testing]
```
Comment (zerolab) [inline tox.ini:67]: I've avoided the extra deps here during the last round of changes as the interactive bit don't need any of the testing requirements per se.

Saying that, we do use dj-database-url which is a testing requirement, so ok

File: README.md [hunk 4]
```diff
@@ -50,7 +50,7 @@ python -m pip install -e .[testing] -U
 
 ```sh
 python -m pip install flit
-flit install
+flit install -s
```
Comment (zerolab) [inline README.md:53]: nice one! (reference for others: `-s` is for symlinking the package so it doesn't need re-installing. Very useful for local dev indeed)

**Review by zerolab — CHANGES_REQUESTED** [review]
A couple of minor changes needed

File: README.md (hunk 4 above)
Comment (tm-kn) [reply README.md:53]: Does anyone develop with just `flit install` at all?

File: tox.ini (hunk 3 above)
Comment (tm-kn) [reply tox.ini:67]: Yeah, it is dj-database-url. Could specify it explicitly if that's better?

File: README.md (hunk 4 above)
Comment (zerolab) [reply README.md:53]: I may know a person... _whistles nonchalantly_

**Comment by zerolab** [comment]
The test failures come from the fact that wagtailmain still doesn't allow Django 5.0. Submitting PR

**Pull request merged**